spec/integration/automation_spec.rb

```diff
@@ -84,5 +86,24 @@
       )
       expect(validated_provider.sent_messages.length).to eq(0)
     end
+
+    it "should not trigger notifications for PMs" do
+      automation.upsert_field!(
+        "watching_categories",
+        "categories",
+        { "value" => [nil] },
+        target: "trigger",
+      )
+
+      topic =
```

Comment:
Can you use a `pm = Fabricate(:private_message_topic)` to create the topic, rather than TopicCreator?